From c342ac98d73fed29a6ee9694d464be24a1ce6942 Mon Sep 17 00:00:00 2001
From: Daniel Mueller <deso@posteo.net>
Date: Fri, 12 Oct 2018 22:23:39 -0700
Subject: [PATCH 2/3] Silently ignore error #2147500037

When importing .ics files from certain sources we always get a pop up
dialog about error 2147500037. It appears that this error may be related
to problems with the time zone present in those events, but the actual
source is unclear. For what it's worth, events seem to be imported just
fine.
However, not only is this error completely unactionable, it is also
annoying and requires additional user interaction. With this patch we
specifically ignore this error and do no longer show a dialog box asking
for the user to confirm.
---
 comm/calendar/base/content/import-export.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/comm/calendar/base/content/import-export.js b/comm/calendar/base/content/import-export.js
index 210ac3..a47cf2 100644
--- a/comm/calendar/base/content/import-export.js
+++ b/comm/calendar/base/content/import-export.js
@@ -176,7 +176,7 @@ function putItemsIntoCal(destCal, aItems, aFilePath) {
             cal.l10n.getCalString("duplicateError", [duplicateCount, aFilePath]),
             window
           );
-        } else if (failedCount) {
+        } else if (failedCount && lastError != 2147500037) {
           cal.showError(
             cal.l10n.getCalString("importItemsFailed", [failedCount, lastError.toString()]),
             window
-- 
2.23.0

